lib/src/_shared/PickerToolbar.tsx

```diff
@@ -72,15 +72,13 @@ const PickerToolbar: React.SFC<PickerToolbarProps> = ({
   penIconClassName,
   toggleMobileKeyboardView,
   toolbarTitle,
-  ...other
 }) => {
   const classes = useStyles();
 
   return (
     <Toolbar
       data-mui-test="picker-toolbar"
       className={clsx(classes.toolbar, { [classes.toolbarLandscape]: isLandscape }, className)}
-      {...other}
```

Comment:
Why removing the spread?

Reply:
It is an internal component that doesn't need to support spreadoing

Reply:
Fair enough